Makefile

```diff
@@ -32,6 +32,10 @@ runfast: run.c
 runomp: run.c
 	$(CC) -Ofast -fopenmp -march=native run.c  -lm  -o run
 
+.PHONY: win64
+win64: 
+	x86_64-w64-mingw32-gcc-win32 -Ofast -D_WIN32 -o run.exe -I. run.c mman-win32.c
```

Comment:
is it common to use underscores like this? why not just -DWIN32

Reply:
_WIN32 is the compiler define, WIN32 is an application level define. I'm not sure if it is required with mingw tbh, it is defined automatically on MSVC.
see: https://stackoverflow.com/questions/662084/whats-the-difference-between-the-win32-and-win32-defines-in-c

Reply:
I see thank you.